e2e-tests/themes/development-runtime/src/gatsby-theme-about/templates/bio.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+export default () => (
+  <>
+    <p data-testid="bio">A bio all about me.</p>
+  </>
+)
```

Comment:
Prettier?

Reply:
How is this not a lint error? :thinking: jsx files ignored?

Reply:
(It doesn't show in the discussion part, but if you look at the actual diff it'll show you that there is no trailing newline)